xwiki-platform-core/xwiki-platform-rest/xwiki-platform-rest-server/src/main/java/org/xwiki/rest/internal/ModelFactory.java

```diff
@@ -1063,4 +1073,19 @@ public JobLog toRestJobLog(Collection<LogEvent> logs, URI self)
 
         return log;
     }
+
+    /**
+     * Check if the given property should be exposed via REST.
+     * @param restProperty the property to be read/written
+     * @param forWrite true if we want to write to the XObject property
+     * @return true if the property is considered public for read or write
+     */
+    private boolean isPublicType(Property restProperty, boolean forWrite)
```

Comment:
I'm not a big fan of making impossible to access the whole model anymore, you might want to use REST to migrate data from one instance to another. At the very least user with PR should be allowed.

Reply:
Yes, I understand. Actually to handle issue it might even be enough to only check for edit rights, or something like that. Update coming soon, I will go for Admin rights in the next attempt, and wait how the discussion sorts out.